zulip/integrations/bridge_with_slack/bridge_with_slack_config.py

```diff
@@ -3,12 +3,17 @@
         "email": "zulip-bot@email.com",
         "api_key": "put api key here",
         "site": "https://chat.zulip.org",
-        "stream": "test here",
-        "topic": "<- slack-bridge",
     },
     "slack": {
         "username": "slack_username",
         "token": "xoxb-your-slack-token",
-        "channel": "C5Z5N7R8A -- must be channel id",
+    },
+    # Mapping between Slack channels and Zulip stream-topic's.
+    # You can specify multiple pairs.
+    "pairs": {
+        "C5Z5N7R8A -- <Slack channel; must be channel id>": {
```

Comment:
Let's just have `C5Z5N7R8A` here, and document what it should be in a comment on the line above; I think there's a good chance some users will have `-- <` in their value otherwise :).

Reply:
Fixed.